Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -1,101 +1,31 @@
+import Checkout
 import UIKit
 import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
-    func updateCountryCode(code:Int)
-    func validateNumber()
+    func phoneNumberIsUpdated(number: String?)
```

Comment:
This syntax might make better if number was non optional. 

Saying
> Phone number is updated

would syntactically imply there is a phone number that was updated. If its nil, the reading becomes more confusing I think?